cmd/deployment-check/Makefile

```diff
@@ -1,5 +1,5 @@
 IMAGE="kuberhealthy/deployment-check"
-TAG="v1.6.2"
+TAG="v1.7.2"
```

Comment:
bumping a minor value should reset the patch to 0 (i.e v1.7.0)

Reply:
done